### DISCUSSION TITLE
Using an alias for my SimpleLogin account?

### DISCUSSION THREAD

**Discussion by SwarfegaGit** [simple-login/app#1854] [Q&A]
I'm trying to think of the consequences of this, and I am thinking this would probably be a bad idea?  Is it even possible to change my simplelogin account to use an alias that is managed by simplelogin itself?

**Comment by nguyenkims** [comment]
To avoid loop and potential issues with having several email forwarding steps, we don't allow to use alias as your mailbox in SL.
